bin/bst-speak.ts

```diff
@@ -63,6 +64,7 @@ Global.initializeCLI().then(
                     console.log("Your token is saved, you can now use this command without providing a token");
                 }
 
+                BstStatistics.instance().record(BstCommand.speak);
```

Comment:
Why do we not capture the node ID? We should always try to record it, but if it has not been setup, then do not send it.